recipe/bld.bat

```diff
@@ -0,0 +1,5 @@
+set INCLUDE_DIRS="%LIBRARY_INC%"
+set LIBRARY_DIRS="%LIBRARY_LIB%"
```

Comment:
I don't know where `%LIBRARY_LIB%` is defined, but this should point to `<conda root>\Library\bin\` where there should be `mfhdf.dll`, `hdf.dll`, and `xdr.dll`.

Also try updating setup.py with the patch below. I'll probably need to release v0.10.1 to get this all working, but for now I want to know what changes are required. It's hard for me to test without a Windows machine that has working compilers.
```diff
diff --git a/setup.py b/setup.py
index 8edc13a..052bc3a 100644
--- a/setup.py
+++ b/setup.py
@@ -133,14 +133,15 @@ if sys.platform == 'win32':
     # Find DLL path
     dll_path = ''
     for p in library_dirs:
-        if path.exists(p + os.path.sep + "HM423M.DLL"):
+        if path.exists(p + os.path.sep + "mfhdf.dll"):
             dll_path = p + os.path.sep
             break
     if dll_path == '':
+        print("library_dirs =", library_dirs)
         raise RuntimeError("Cannot find required HDF4 DLLs -- check LIBRARY_DIRS")
 
 if sys.platform == 'win32':
-    libraries = ["hm423m", "hd423m", "xdr_for_dll" ]
+    libraries = ["mfhdf", "hdf", "xdr" ]
 elif _use_hdf4alt(library_dirs):
     libraries = ["mfhdfalt", "dfalt"]
 else:
@@ -172,7 +173,7 @@ _hdfext = Extension('pyhdf._hdfext',
                     )
 
 if sys.platform == 'win32':
-    data_files = [("pyhdf", [dll_path + x for x in ["HM423M.DLL", "HD423M.DLL"]])]
+    data_files = [("pyhdf", [dll_path + x for x in ["mfhdf.dll", "hdf.dll"]])]
 else:
     data_files = []
 
```

Reply:
I was able to build and test pyhdf successfully in AppVeyor/Miniconda: https://github.com/fhs/pyhdf/pull/25

Reply:
> I don't know where %LIBRARY_LIB%

`%LIBRARY_LIB%` expands to `%PREFIX%\Library\lib` where the `.lib` files are. Based on what you mentioned we need `%PREFIX%\Library\bin`, that is where the `.dlls` are, thanks!